src/lib/run_environment.ml

```diff
@@ -94,10 +94,37 @@ module Tool = struct
   end
 end
 
+module Make_fun = struct
+  module Requirement = struct
+    type t = [
+      | `Processors of int
+      | `Internet_access
+      | `Memory of [ `GB of float | `Decent | `Big ]
```

Comment:
Having two ambiguous options like this is just asking for trouble. Both are subjective and uninformative.